.travis.yml

```diff
@@ -1,10 +1,23 @@
 language: python
-python:
-    - 2.7
-install:
-    - "pip install tox"
-env:
-    - TOX_ENV=django18
-    - TOX_ENV=django111
-script:
-    - tox -e $TOX_ENV
+matrix:
+  include:
+    - install:
+      - git clone https://github.com/edx/devstack.git
+      - (cd devstack; sed -i 's/:cached//g' ./docker-compose-host.yml)
+      - (cd devstack; make dev.clone)
+      script:
+        - (cd devstack; DEVSTACK_WORKSPACE=$PWD/.. docker-compose -f docker-compose.yml -f docker-compose-host.yml run -v $PWD/..:/edx-sga lms /edx-sga/run_sga_tests.sh)
```

Comment:
I think I understand why you're mounting that directory to `/edx-sga` (so you know where it is within the int. test shell script), but does it not make more sense to copy the working directory to `/edx-sga` as the first step in `install`? (1) This kind of obscures the fact that we're putting the edx-sga repo at that location, and (2) by the time you're doing this mount, that directory has all of the cloned edx repos in it as well, right? I don't think we need those. Might be misreading this, though